applications/Covid Tracker/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width"> <!--to create a responsive site--> 
+        <meta name="description" content="Tracking covid cases around me"> 
+        <meta name="keywords" content="covid19 cases, tracking covid cases, covid tracker"> <!--for SEO--> 
+        <meta name="author" content="Rendell">
+        <title>Covid Tracker | Home</title>
+        <link rel="stylesheet" href="./css/style.css">
+    </head>
+    <Body>
+        <header>
+            <div class="container">
+                <div id="branding">
+                    <h1>Covid Tracker</h1>
+                </div>
+                <nav>
+                    <ul>
+                        <li><a href="index.html">Home</a></li>
+                        <li><a href="about.html">About Us</a></li>
+                        <li><a href="donate.html">Donate</a></li>
+                    </ul>
+                </nav>
+            </div>
+        </header>
+        <section id="showcase">
+            <div class="container">
+                <h1>Let's Tackle Covid Together</h1>
```

Comment:
Since you already have an `<h1>` in the header, you could probably make this an `<h2>`.